CITATION.cff

```diff
@@ -460,10 +478,12 @@ authors:
     family-names: Gensollen
     website: https://nicolasgensollen.github.io/
     affiliation: Inria, Paris, France
+    orcid: https://orcid.org/0000-0001-7199-9753
```

Comment:
@nicolasgensollen

is that your ORCID?

Reply:
@Remi-Gau Yes it is ! 👍 Thanks for this ! 🙏